src/fiatconnect/fiatAccountSchemas/ifscAccount.ts

```diff
@@ -0,0 +1,58 @@
+import { FiatAccountSchema, FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getObfuscatedAccountNumber } from 'src/fiatconnect'
+import { INSTITUTION_NAME_FIELD } from 'src/fiatconnect/fiatAccountSchemas'
+import { FiatAccountFormSchema } from 'src/fiatconnect/fiatAccountSchemas/types'
+import i18n from 'src/i18n'
+
+export const getIfscAccountSchema = (implicitParams: {
+  country: string
+  fiatAccountType: FiatAccountType
+}): FiatAccountFormSchema<FiatAccountSchema.IFSCAccount> => {
+  const accountNumberValidator = (input: string) => {
+    const isValid = /^[0-9]+$/.test(input)
```

Comment:
no length LB/UB?

Reply:
I believe this is dependent on banks. I found this https://www.bankindia.org/2020/06/all-banks-account-number-digits-in-india.html, although not sure if this list is comprehensive.